curl/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import shutil
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+
+
+class CurlConan(ConanFile):
+    name = 'curl'
+    version = '7.28.1'
+    url = 'https://github.com/wsbu/conan-packages'
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    license = 'see COPYING file'
+
+    ARCHIVE = '{0}-{1}.tar.gz'.format(name, version)
+    SOURCE_DIR = '{0}-{1}'.format(name, version)
+
+    options = {
+        'shared':     [True, False],
+        'ldap':       [True, False],
+        'ldaps':      [True, False],
+        'with_axtls': [True, False],
+        'with_ssl':   [True, False],
+        'with_zlib':  [True, False]
+    }
+    default_options = 'shared=True', \
+                      'ldap=False', \
+                      'ldaps=False', \
+                      'with_axtls=False', \
+                      'with_ssl=True', \
+                      'with_zlib=True'
+
+    def requirements(self):
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11@conan/stable")
+        if self.options.with_ssl:
+            self.requires("OpenSSL/1.0.2n@conan/stable")
```

Comment:
Can you switch this to 1.0.2o? Without that change, you'll run into the same problem here that we're seeing with mosquitto: https://ci.redlion.net/viewLog.html?buildId=2390&buildTypeId=MqttMosquitto_Conan&tab=buildLog&_focus=458
And for some reason, upstream conan guys fixed it for OpenSSL v1.0.2o :) (I tested it on my machine just now)